machines/settings.ts

```diff
@@ -126,6 +129,20 @@ export const settingsMachine = model.createMachine(
           },
         },
       },
+      injiToutGuide: {
+        on: {
+          INJI_TOUR_GUIDE: {
+            target: 'showInjiTourGuide',
+          },
+        },
+      },
+      showInjiTourGuide: {
+        on: {
+          BACK: {
+            target: 'idle',
+          },
+        },
+      },
```

Comment:
@PuBHARGAVI Can you review the changes in the state-machine's state change for showing the tour-guide from settings?